lab-judy/test/gallery-service-test.js

```diff
@@ -0,0 +1,185 @@
+'use strict';
+
+describe('testing gallery service', function(){
+
+  // beforeEach mocks the demoApp module
+  //            mocks the service
+
+  beforeEach(() => {
+    angular.mock.module('demoApp');
+    angular.mock.inject((authService, galleryService, $httpBackend, $window, $rootScope) => {
+      this.authService = authService;
+      authService.setToken('1234');
+
+      this.galleryService = galleryService;
+      this.$window = $window;
+      this.$rootScope = $rootScope;
+      this.$httpBackend = $httpBackend;
+    });
+  });
+
+  afterEach(() => {
+    this.authService.setToken(null);
+    this.$window.localStorage.clear();
+  });
+
+  //POST /api/gallery
+  describe('testing POST galleryService.createGallery', () => {
+    it('should return a gallery', () => {
+      let galleryData = {
+        name: 'exampleGallery',
+        desc: 'memories from my beach adventure',
+      };
+
+      let headers = {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+        Authorization: 'Bearer 1234',
+      };
+
+      this.$httpBackend.expectPOST('http://localhost:3000/api/gallery', galleryData, headers)
+      .respond(200, {_id: '5678', username: 'slugbyte',  name: galleryData.name, desc: galleryData.desc, pics: []});
+
+      this.galleryService.createGallery(galleryData);
```

Comment:
You aren't testing anything here -- you want to 'then' off of your method and test the expected value resolved from the method with an expect statement within the then block.